tools/traffic_simulator/traffic_simulator.py

```diff
@@ -37,6 +37,22 @@ def worker_func(args):
         if expected != res:
             print("# failed!!! key= " + key_format.format(index=key_index) + " expected= " + str(expected) + " res= " + str(res))
             return -1
+        if with_compaction:
+            info = redis_client.execute_command('TS.INFO', key_format.format(index=key_index))
+            response = dict(zip(info[::2], info[1::2]))
+            if not 'rules' in response:
+                print("# failed!!! key= " + key_format.format(index=key_index) + " No rules")
+                return -1
+            if len(response['rules']) != 3:
+                print("# failed!!! key= " + key_format.format(index=key_index) + " Number of rules isn't equal 3, rules = " + str(response['rules']))
+                return -1
+        res = redis_client.execute_command('TS.QUERYINDEX', 'index=' + str(key_index))
+        n_res = 1
+        if with_compaction:
+            n_res += 3
```

Comment:
can we add 'n_res += 3' to the end of the first 'if with_compaction:' so we will have only 1 if?
and 'n_res = 1' before that first if so it will not overwrite it